__tests__/app.test.js

```diff
@@ -220,4 +220,39 @@ describe("News app",()=>{
             })
         })
     })
+    describe('GET /api/articles/:article_id/comments', () => {
```

Comment:
One more test cases to consider: What should we respond with for an article that has no comments?